pom.xml

```diff
@@ -2534,7 +2534,6 @@
             <checkMultipleScalaVersions>true</checkMultipleScalaVersions>
             <failOnMultipleScalaVersions>true</failOnMultipleScalaVersions>
             <recompileMode>incremental</recompileMode>
-            <useZincServer>true</useZincServer>
```

Comment:
Is this a default now? I guess we'll look at the build logs to ensure zinc is actually used

Reply:
The [`useZincServer` has been removed](https://github.com/davidB/scala-maven-plugin/pull/321/files#diff-7d0815d86a5ea5598e1ee37c18a5592e90820c22ea61202bd83ae0dba066225dL83-L84) and the [incremental feature is enable by default](https://github.com/davidB/scala-maven-plugin/commit/0f01f2fe37aaadbefb02fb713d469dbd10acf8b1), in scala-maven-plugin 4.X. and the arg [recompileMode](https://github.com/davidB/scala-maven-plugin/blob/d7d19a2e98e7661d30fadb726fb0e91177eee5b6/src/main/java/scala_maven/ScalaCompilerSupport.java#L45-L50) is used to enable or disable the increamtal compiler.